components/molecules/DrawerMenuItem.tsx

```diff
@@ -1,16 +1,16 @@
 import { ListItemIcon, ListItemText, MenuItem } from '@mui/material';
-import React from 'react';
-import { SupportedLocale } from '../../@types';
+import React, { useContext } from 'react';
 import { LocalizedLink } from '../atoms/LocalizedLink';
+import { LocaleContext } from './TranslationsWrapper';
 
 type Props = {
   icon: React.ReactNode
   label: string
-  locale: SupportedLocale
   path: string
 };
 
-export const DrawerMenuItem = ({ icon, label, locale, path }: Props) => {
+export const DrawerMenuItem = ({ icon, label, path }: Props) => {
+  const locale = useContext(LocaleContext).locale;
```

Comment:
Note to ask later: Is it possible to write it like this?
(I genuinely don't know because I can't try it right now)

```ts
  const { locale } = useContext(LocaleContext);
```

Reply:
Agreed to change this